iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/utils/IcebergHiveCachedClientPool.java

```diff
@@ -107,7 +112,13 @@ public IcebergHiveCachedClientPool(Configuration conf, Map<String, String> prope
   @VisibleForTesting
   HiveClientPool clientPool() {
     Key key = extractKey(properties.get(CatalogProperties.CLIENT_POOL_CACHE_KEYS), conf);
```

Comment:
when enabled , the `CatalogProperties.CLIENT_POOL_CACHE_KEYS` should not be null and must cantain `ugi` or `user_name`

Reply:
I see. Thanks for the explanation.